src/lib/errors/base-http-error.js

```diff
@@ -1,12 +1,45 @@
-class BaseHttpError extends Error {
-  constructor(message, status) {
+// Stub for extends from error in ES6 classes.
+// NOTE/TODO: Remove it and do extends from Error when this behavior will be fixed.
```

Comment:
This is the PR to sort out all problems now, no stubs (stubs can be present only in testing env)
So fix all error todo's now.